eventmesh-sdks/eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/grpc/consumer/EventMeshGrpcConsumer.java

```diff
@@ -85,7 +85,8 @@ public EventMeshGrpcConsumer(final EventMeshGrpcClientConfig clientConfig) {
     }
 
     public void init() {
-        this.channel = ManagedChannelBuilder.forAddress(clientConfig.getServerAddr(), clientConfig.getServerPort()).usePlaintext().build();
+        this.channel = ManagedChannelBuilder.forAddress(clientConfig.getServerAddr(), clientConfig.getServerPort()).enableRetry().usePlaintext()
```

Comment:
> Doesn't gRPC generally supports retry by default? Or rather, it's not like that, we must manually enable retry?
> 
> gRPC一般不是默认支持重试吗？还是说并非如此，必须手动开启重试？

This can be deleted.